.github/workflows/ci.yml

```diff
@@ -238,10 +238,52 @@ jobs:
         timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
         run: bash _ci/test.sh ${{ matrix.testProfile }}
 
+  e2e_javaruntime_tests:
+    name: "E2E AIO Tests : ${{ matrix.testName }} - ${{ matrix.java }}"
```

Comment:
1. Pls rename it to `E2E AIO Smoke Tests (Java: x)`
2. the `${{ matrix.testName }}` is not used anymore